ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -919,6 +932,16 @@ vector<double> OwInterface::getEndEffectorFT () const
   return m_end_effector_ft;
 }
 
+std::vector<bool> OwInterface::getSubsystemFault(string subsystem_name) const
+{
+  return m_fault_hierarchy->getSubsystemStatus(subsystem_name);
+}
+
+std::vector<std::string> OwInterface::getActiveFaults (std::string subsystem_name) const
+{
+  return m_fault_hierarchy->getActiveFaults(subsystem_name);
+}
```

Comment:
Try to be consistent about your namespacing. This file has `using std::vector;` and `using std::string;`, so you could just omit the `std::`.